pkgs/top-level/all-packages.nix

```diff
@@ -13387,8 +13390,12 @@ let
 
   vym = callPackage ../applications/misc/vym { };
 
-  w3m = callPackage ../applications/networking/browsers/w3m {
+  w3m = callPackage ../applications/networking/browsers/w3m { };
+
+  w3m-pure = w3m.override {
```

Comment:
Use `self.w3m.override`, otherwise this would add a security issue when we fix `w3m`.